### ISSUE TITLE
Augmented Face issues: stretched & not aligned well.

### ISSUE THREAD

**Issue by dkmoon** [google-ar/arcore-android-sdk#1283]
### SPECIFIC ISSUE ENCOUNTERED

The Augmented Face example seems working weird on some device.
I've built the code as-is and found two issues.

1) Background camera image is stretched toward longer side. Vertically when in the portrait mode, and horizontally when in the landscape mode.  Please check out screenshots below.

2) Models and background camera image are not aligned very well. Please see the freckle texture in the screenshots. It's not aligned and actually bigger than the recognized face.

Portrait image: https://drive.google.com/file/d/1PmoyqsIqJ3hAmh5GowCr75aMXFXg0ePA/view?usp=sharing
Landscape image: https://drive.google.com/file/d/1OhqsknosFcP3-fL9SEMFTtRt5SUEeTGv/view?usp=sharing

### VERSIONS USED
- Android Studio: Arctic snow 2020.3.1 patch 1
- ARCore SDK for Android: com.google.ar:core:1.26.0
- Device manufacturer, model, and O/S: Samsung SM-J730K
- Google Play Services for AR (ARCore):  1.26.212010303
  On Windows, use: `adb shell pm dump com.google.ar.core | findstr /i "packages: versionName"`
  On macOS, use: `adb shell pm dump com.google.ar.core | egrep -i versionName\|packages:`
- Output of `adb shell getprop ro.build.fingerprint`:  samsung/j7y17ltektt/j7y17ltektt:9/PPR1.180610.011/J730KKKS5CUD1:user/release-keys

### STEPS TO REPRODUCE THE ISSUE
 1. clone the repo
 1. open the augmented_face_java example in Android Studio.
 1. Build and run on Samsung J7 (SM-J730K)

**Comment by devbridie** [comment]
I can't see the images you've linked; could you adjust the sharing settings?

**Comment by devbridie** [comment]
Looks like a 4:3 image stretched vertically in portrait and horizontally in landscape to 16:9.

**Comment by dkmoon** [comment]
Great to hear you figured out what causes the problem. Then, how can I fix it? As I said, the code is exactly the same as one in the github repo. I tried to track down but it seems ARCore's native implementation generates OpenGL texture in a wrong aspect ratio on the device in question. (I read a texture after background camera drawing by glReadPixels, and could confirm the texture generated is already stretched.) I also checked CameraConfigFilter but it reports it's using 1920x1080 resolution.
